.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "third_party/benchmark"]
+	path = third_party/benchmark
+	url = https://github.com/google/benchmark
```

Comment:
Is this actually used? You have `google-benchmark` in Python test requirements already.

A source dependency on the C++ library could be added to shortfin as needed, probably via FetchContent here: https://github.com/nod-ai/shark-ai/blob/4eac34e1455f6518d5813299f712137af6a96573/shortfin/CMakeLists.txt#L326-L340

Reply:
I wrongly left the pip package dependency. It is removed now. Unfortunately, the script to compare benchmark results is not a part of the pip package. This script is used in the python test.
See https://github.com/google/benchmark/blob/main/docs/tools.md#comparepy
That is why the submodule is used.